scripts/setup.py

```diff
@@ -411,14 +411,18 @@ def clean_up(files: list[str]):
 	)
     validate_parameters(project)
 
-    options = {
-		'none' : 'none',
-		'kebab (kebab-case)' : 'kebab',
-		'snake (snake_case)' : 'snake',
-		'pascal (PascalCase)' : 'pascal'
-	}
-    choice = enquiries.choose('Choose default json_serializable.field_rename: ', options.keys())
-    project.json_serializable = JsonSerializable(options[choice])
+    if os.environ.get("CI") != "true":
+        options = {
+            'none': 'none',
+            'kebab (kebab-case)': 'kebab',
+            'snake (snake_case)': 'snake',
+            'pascal (PascalCase)': 'pascal'
+        }
+        choice = enquiries.choose('Choose default json_serializable.field_rename: ', options.keys())
+        project.json_serializable = JsonSerializable(options[choice])
+    else:
+        # Skip enquiries on CI
+        project.json_serializable = JsonSerializable('snake')
 
     print(f"=> 🐢 Staring init {project.new_project_name} with {project.new_package}...")
```

Comment:
@luongvo typo `Staring` -> `Starting` 🤔

Reply:
@doannimble oops, it's not from my PR but fixed 🐰